packages/command/src/sfdxFlags.ts

```diff
@@ -625,11 +641,15 @@ function validateCustomFlag<T>(key: string, flag: flags.Any<T>): flags.Any<T> {
   return flag;
 }
 
+function isBuiltin(flag: object): flag is flags.Builtin {
+  return hasString(flag, 'type') && flag.type === 'builtin';
+}
+
 /**
  * Builds flags for a command given a configuration object.  Supports the following use cases:
- *     1. Enabling common SFDX flags. E.g., { verbose: true }
- *     4. Defining typed flags. E.g., { myFlag: Flags.array({ char: '-a' }) }
- *     4. Defining custom typed flags. E.g., { myFlag: Flags.custom({ parse: (val) => parseInt(val, 10) }) }
+ * 1. Enabling common SFDX flags. E.g., { verbose: true }
+ * 4. Defining typed flags. E.g., { myFlag: Flags.array({ char: '-a' }) }
```

Comment:
Should all of these be `1.`?

Reply:
👍 Changed to dashes, I think it's more or less a list